lighthouse-core/test/audits/dobetterweb/uses-http2-test.js

```diff
@@ -5,21 +5,31 @@
  */
 'use strict';
 
+const Runner = require('../../../runner.js');
 const UsesHTTP2Audit = require('../../../audits/dobetterweb/uses-http2.js');
 const assert = require('assert');
+const networkRecordsToDevtoolsLog = require('../../network-records-to-devtools-log.js');
 
 const URL = 'https://webtide.com/http2-push-demo/';
 const networkRecords = require('../../fixtures/networkRecords-mix.json');
 
 /* eslint-env jest */
 
 describe('Resources are fetched over http/2', () => {
+  function getArtifacts(networkRecords, finalUrl) {
+    // networkRecords-mix.json is an old network request format, so don't verify round-trip.
+    const devtoolsLog = networkRecordsToDevtoolsLog(networkRecords, {skipVerification: true});
```

Comment:
using a real networkRecord as input has a few problems with verification of the output, namely that we've already [snipped a bunch of circular references and replaced them with strings](https://github.com/GoogleChrome/lighthouse/blob/fbcb1e3ea3aa05b0cfdf87ddbc8155bb6fe1783f/lighthouse-core/test/fixtures/networkRecords-mix.json#L7) and we've switched a few properties around since that old version of `NetworkRecorder`. The subset needed for this test generates correctly, so it seemed better to just not verify rather than edit the json file.